docs/tutorial.rst

```diff
@@ -190,7 +190,8 @@ BASH_COMPLETIONS   ``[] or ['/etc/...']``      This is a list of strings that
                                                specifies where the BASH
                                                completion files may be found.
                                                The default values are platform
-                                               dependent, but sane.
+                                               dependent, but sane.  To specify an alternate list,
```

Comment:
Nope that's fine

On Wed, Apr 1, 2015, 8:04 PM Steven Silvester <notifications@github.com>
wrote:

> In docs/tutorial.rst
> <https://github.com/scopatz/xonsh/pull/179#discussion_r27626415>:
>
> > @@ -190,7 +190,8 @@ BASH_COMPLETIONS   ``[] or ['/etc/...']``      This is a list of strings that
> >                                                 specifies where the BASH
> >                                                 completion files may be found.
> >                                                 The default values are platform
> > -                                               dependent, but sane.
> > +                                               dependent, but sane.  To specify an alternate list,
>
> Sorry, my editor is set to remove trailing spaces, do you want me to
> revert these other changes?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scopatz/xonsh/pull/179/files#r27626415>.
>